jax_unirep/evotuning.py

```diff
@@ -185,15 +260,42 @@ def step(i, state):
         params = get_params(state)
         g = grad(evotune_loss)(params, x, y)
         state = update(i, g, state)
+
         return state
 
     state = init(params)
-    from time import time
 
     for i in range(n):
+
+        logger.info(f"Starting epoch {i + 1}")
+
         for x, y in zip(xs, ys):
             state = step(i, state)
-            params = get_params(state)
+
+            # for debugging only
+            # params = get_params(state)
+            # logger.info(f"Shape of y: {(len(y), len(y[0]), len(y[0][0]))}")
+            # logger.info(softmax(vmap(partial(predict, params))(x)))
```

Comment:
I think we can set this to logging.debug level, and hence turn it on globally when necessary. It's a useful piece of information to have when debugging.